sql/core/src/main/scala/org/apache/spark/sql/execution/DataSourceScanExec.scala

```diff
@@ -102,6 +102,7 @@ case class RowDataSourceScanExec(
     requiredSchema: StructType,
     filters: Set[Filter],
     handledFilters: Set[Filter],
+    aggregation: Aggregation,
```

Comment:
are you going to add this feature to DS v1? I think only file source needs it, which is `FileSourceScanExec`

Reply:
I see, you add it only for display purposes. We can probably skip it for now, as I don't think v2 sources that do v1 fallback will support aggregate pushdown.

Reply:
Will do